omod/src/main/java/org/openmrs/module/spa/SpaDirectoryResolver.java

```diff
@@ -75,6 +90,13 @@ private static void resolveDirectory(String spaDirectory) {
             return;
         }
 
+        if(spaDirectory.equals(BUNDLED_FRONTEND_DIRECTORY)){
+            String openmrsRootPath = servletContext.getRealPath("/");
+            File file = new File(openmrsRootPath, "WEB-INF" + File.separator + spaDirectory);
+            SpaDirectoryResolver.spaDirectory.set(Paths.get(file.toURI()).normalize().toAbsolutePath().toString());
```

Comment:
Why are we using both `File` and `Paths` here? Just use one or the other (`Paths.get(openmrsRootPath, 'WEB-INF', spaDirectory).toAbsolutePath().toString()`)